src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -297,12 +297,28 @@ public void testFuncotatorWithoutValidatingResults(final String dataSourcesPath,
         runCommandLine(arguments);
     }
 
+    private void validateFuncotationsOnVcf(final Iterable<VariantContext> vcfIterable, final String[] funcotationFieldNames) {
```

Comment:
Do you have at least one test case that fails without this patch, and passes with it?

Reply:
A test case for what specifically?  This PR was created to increase the speed of annotation - I don't have a test that goes slowly without and faster with this patch.